src/components/Articles/Articles.vue

```diff
@@ -47,21 +53,40 @@ export default class Articles extends Mixins(CommonMixin) {
   // Whether components have changes
   private companyProvisionsChanges: boolean
 
+  private isEditingCompanyProvisions = false
```

Comment:
This is the same name as the company provisions component event. While this may not be causing a conflict, it may result in confusion.
How about renaming the event `isEditMode` (same as some of the other components)?

Reply:
no, I meant to check if the component was left without finish editing. I just followed what we have in YourCompany component: `isEditingNames`, `isEditingType` etc.

Reply:
+1 to Severins comment. 
In this case it makes sense to rename the component event to something more generic, as we KNOW that event belongs to the component, where the local property needs to be specific.

Reply:
The local variable name is fine, but I think we should have consistent event names.

Reply:
>The local variable name is fine, but I think we should have consistent event names.

Do you guys think I should change the name? I followed what we have for other components like in YourCompany.

Reply:
Yes, please change the event name (not the variable name).